cloudfront/wellcomelibrary.org/edge-lambda/src/paths.ts

```diff
@@ -0,0 +1,28 @@
+export type GetBNumberResult = string | Error;
```

Comment:
Boring thought: you could make this look even more like scala by creating a generic for reuse everywhere:
```typescript
type Try<T> = T | Error;
export type GetBNumberResult = Try<string>;
```

Reply:
I did think about that, but ended up not as I feel it leads towards "use a library for functional stuff" which i'd like to avoid here at least.